trojsten/polls/models.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Question(models.Model):
+    text = models.CharField(max_length=1000)
+    created_date = models.DateTimeField(default=timezone.now)
+    deadline = models.DateTimeField(default=datetime(2020, 12, 31, 23, 59, 59))
+
+    def __str__(self):
+        return self.text
+
+
+class Answer(models.Model):
+    text = models.CharField(max_length=1000)
+    question = models.ForeignKey("Question", on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f'Answer "{self.text}" to question "{str(self.question)}"'
+
+
+class Vote(models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    created_date = models.DateTimeField(default=timezone.now)
+    answer = models.ForeignKey("Answer", on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f'Vote for "{str(self.answer)}"'

```

Comment:
Bohuzial v produkcii mame stale python 3.5, takze format stringy nie su este podporovane. Snazime sa to zmenit ale nie som si isty kedy to bude. Takze ak s tymto PR nechceme cakat do neurcita, tak ich zatial nepouzivaj.

Reply:
Anglicke texty pls obal translate funkciou a pridaj prekaldy.

Reply:
Ako sa kompiluju preklady? Podla wiki na githube mam spustit django_admin.py, taky vsak neviem najst

Reply:
django_admin.py maz v path. Ale namiesto django_admin.py sa da pouzit aj manage.py len si daj pozor aby si bol v tom spravnom adresari (v tom co pise nasa wiki). Mozno musis volat ../manage.py. Ale django_admin.py mozes vpohode volat ak si vo virtualenve kde mas nainstalovane django.